src/components/share/share.js

```diff
@@ -48,15 +81,26 @@ class ShareControl extends BaseComponent {
    * @returns {React$Element} component element
    * @memberof ShareControl
    */
-  render(): React$Element<any> {
+  render(): React$Element<any> | void {
+    const {embedUrl, enable, shareUrl} = this.props.config;
+    if (!(enable && shareUrl && embedUrl)) {
```

Comment:
why not shareUrl || embedUrl - could only one be configured from app no?

Reply:
both options must be valid and available